blog/models.py

```
@@ -44,7 +44,10 @@ class PostImage(models.Model):
 
 
 class Pin(models.Model):
```

Ruff (0.15.6)

[warning] Model does not define `__str__` method

(DJ008)